src/mmw/js/src/core/views.js

```diff
@@ -198,17 +193,22 @@ var MapView = Marionette.ItemView.extend({
         // supported, nothing more needs to be done since the map has
         // already been centered.
         if (navigator.geolocation) {
-            var options = {
+            var geolocationOptions = {
```

Comment:
Yes. At one point I was passing options into the initialize function, so I wanted to make it clearer, but it's not necessary anymore.